### PR TITLE
Create printf.c

### LINKED ISSUE #91: [tutorial] Add printf function in common built-in functions
**Description of the tutorial**
I want add printf function as well is common built-in functions folder.

**Do you want to be assigned to work on this?**
(mark [x])
- [x] YES
- [ ] NO

**Optional: Any additional info you want to add**
 add this under hacktoberfest

### PR DESCRIPTION
**Pull request by hamzabohra**
### 2. Description
This tutorial is about printf function in c. This tutorial contains a tutorial, code with comments and a challenge.

### 3. Fill in checklist by marking [x]

- [x] I've read the CONTRIBUTING.md
- [x] I was assigned to this issue
- [ ] My code is formatted using required linting
- [x] I've run my code locally and checked it works

### CODE REVIEW

**Comment by hamzabohra** [comment]
Please add this under hacktoberfest.

**Comment by hamzabohra** [comment]
bro plz merge my pr plz !!

**Comment by hamzabohra** [comment]
only 1 pr is remaining for me plzzz!!

File: 11-common-built-in-functions/printf.c [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  printf("Float f = %f \n", f);
+  // To print double type, %lf is used.
+  printf("Double d = %lf \n", d);
+  // You can do some operation printf function and result will be displayed.
+  printf("a + d = %lf \n", a+d);
+  return 0;
+}
+/*
+--------------------------------------------------------------------------------------------------------------------------------------
+Challange: Take input from user in variable a and b, print number of elements present in each variable and then print sum of number of
```
Comment (sabderemane) [inline 11-common-built-in-functions/printf.c:42]:
```suggestion
Challenge: Take input from user in variable a and b, print number of elements present in each variable and then print sum of number of
```

**Review by sabderemane — COMMENTED** [review]
I think it's ok, just a little typo :)

**Comment by funbeedev** [comment]
@hamzabohra Hi, I edited the format of the tutorial to make it a bit easier to read. Thanks for your contribution!

**Pull request merged**